app/src/main/kotlin/maru/app/MaruApp.kt

```diff
@@ -248,10 +248,21 @@ class MaruApp(
           syncStatusProvider = syncStatusProvider,
         )
       } else {
+        val elPayloadValidatorHandler =
+          createElNodeHandlers(
+            mapOf("payload-validator" to config.validatorElNode.engineApiEndpoint),
+            beaconGenesisConfig,
+          )
+        val elPayloadValidatorNewBlockHandler =
+          NewBlockHandlerMultiplexer(elPayloadValidatorHandler + metadataCacheUpdaterHandlerEntry)
```

Comment:
`metadataCacheUpdaterHandlerEntry` needs to be updated after the validation, so it should be in the same map as `elFollowerHandlersMap` and I think this means that we can move the instantiation of the follower handlers out of both branches, same as it used to be